src/test/ui/destructuring-assignment/tuple_struct_destructure_fail.stderr

```diff
@@ -31,6 +31,8 @@ LL | struct TupleStruct<S, T>(S, T);
 ...
 LL |     TupleStruct(_) = TupleStruct(1, 2);
    |     ^^^^^^^^^^^^^^ expected 2 fields, found 1
+   |
+   = help: use `..` to ignore unmentioned fields
```

Comment:
I think it should also mention another way which is to use `_, _` to fill out all the fields, this is more explicit and I think should be more preferable.